Makefile

```diff
@@ -328,7 +328,7 @@ nxOMSGenerateInventoryMof:
 
 nxOMSPlugin:
 	rm -rf output/staging; \
-	VERSION="2.9"; \
+	VERSION="3.0"; \
```

Comment:
can you rebase to master branch? the latest version should be 2.10. this one should use 2.11